<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCRForReadClass</class>
 <widget class="QMainWindow" name="OCRForReadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCRForRead</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>panda.ico</normaloff>panda.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Stop.ico</normaloff>Stop.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_browse">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>启动摄像机</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelshow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>750</width>
      <height>750</height>
     </rect>
    </property>
    <property name="text">
     <string>Image display area</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>识别</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Start.ico</normaloff>Start.ico</iconset>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>-40</y>
      <width>571</width>
      <height>871</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ocrforread.qrc"/>
  <include location="D:/Project/OCR_For_Show_V0.7/STSTest_1101B/WindowsForFun.qrc"/>
 </resources>
 <connections/>
</ui>
